### ISSUE TITLE
DateTimePicker - How can I change style of label?  ie bold?

### ISSUE THREAD

**Issue by mikezimm** [pnp/sp-dev-fx-controls-react#568]
### Category
[ ] Enhancement

[ ] Bug

[ X ] Question

### Version   
"@pnp/spfx-controls-react": "1.16.0",

### Expected / Desired Behavior / Question
I would like to make the Title above the date picker Bold (red arrows) to be consistent with my other text fields (black arrow):

![image](https://user-images.githubusercontent.com/49648086/82765580-0929e180-9de6-11ea-8075-b558b9fb43fb.png)

### Steps to Reproduce
If I manually add it in debugger mode on the label as shown below (yellow highlight), the label is bold:
![image](https://user-images.githubusercontent.com/49648086/82765788-b7825680-9de7-11ea-8d7f-2cfb0c89ab49.png)

Since this control doesn't have a styles property, I've tried this in my scss file but it does not seem to have any impact.  Is my css incorrect or is it not possible with this control?

Note in the previous screenshot, I also highlighted in blue the classes that I have on the control.

![image](https://user-images.githubusercontent.com/49648086/82765740-37f48780-9de7-11ea-86bc-e27a70412ca9.png)

**Comment by AJIXuMuK** [comment]
Hi @mikezimm.
Currently there are no properties for that.
But it definitely a valid point for enhancement.
As a temporary workaround I can suggest to add such a CSS to your page:
```css
[class^=dateTimePicker_] > .ms-Label {
  font-weight: 700
}
```

**Comment by mikezimm** [comment]
@AJIXuMuK , Thanks for the info... worked like a charm!
I will try to get that to work for the People Picker and Choice field as well (screenshot number 2).

For educational purposes, could you verify if this is what it means in layman's terms?  I never would have came up with that on my own but I was able to try and translate it as follows using w3Schools:

1. The **:global** looks for everything on the page, except I wrapped my under **.projectPage** so I don't know if that has any impact.
2. the **[class^=dateTimePicker_]** looks for anything where the className _**begins**_ with dateTimePicker_ - which helps for the unique class alias' for my webpart.
3. the **> .ms-Label {** then finds any ms-Label classes directly under the **dateTimePicker_** classes.

My only question is should I move it outside of my **.projectPage** class?  It works, but I'm just curious.

Thanks!

![image](https://user-images.githubusercontent.com/49648086/85300811-0833b080-b475-11ea-8145-4fa744666301.png)

![image](https://user-images.githubusercontent.com/49648086/85300791-02d66600-b475-11ea-8d8d-c37758900a08.png)

[css_selector filters on w3schools](https://www.w3schools.com/cssref/css_selectors.asp)
[attribute] | [target] | Selects all elements with a target attribute
-- | -- | --
[attribute=value] | [target=_blank] | Selects all elements with target="_blank"
[attribute~=value] | [title~=flower] | Selects all elements with a title attribute containing the word "flower"
[attribute\|=value] | [lang\|=en] | Selects all elements with a lang attribute value starting with "en"
[attribute^=value] | a[href^="https"] | Selects every <a> element whose href attribute value begins with "https"
[attribute$=value] | a[href$=".pdf"] | Selects every <a> element whose href attribute value ends with ".pdf"
[attribute*=value] | a[href*="w3schools"] | Selects every <a> element whose href attribute value contains the substring "w3schools"

**Comment by mikezimm** [comment]
@AJIXuMuK ,
I did some trial and error and have a follow-up:

I was able to get the labels on my dropdown and date picker to work nesting those references in the same :global setting.

However, I noticed that for my class for the PeoplePicker, using the **class|** filter does not work where the **class*=** does work.  is it because I have multiple classes and it takes the entire list of classes as one string?  Also, what would be the difference between 'begins with' and ' starting with' for the **| and ^** filters?  Just curious :)  Thanks again for the reply... really helped!

![image](https://user-images.githubusercontent.com/49648086/85306412-70d25b80-b47c-11ea-9f18-b7b7f66a6acd.png)

![image](https://user-images.githubusercontent.com/49648086/85306648-c7d83080-b47c-11ea-953b-22683b144538.png)

**Comment by AJIXuMuK** [comment]
Hi @mikezimm,

> The :global looks for everything on the page, except I wrapped my under .projectPage so I don't know if that has any impact.

`:global` in scss tells that class names in the section should not be prefixed when compiling to CSS. It means you can reference classes from the page in the `:global` section.
And that's fine to have it included inside `projectPage` to not impact styling of other web parts that can use the same components.

>  is it because I have multiple classes and it takes the entire list of classes as one string?

Yes, the rule of `^` is "attribute begins with". If you have more than one class name it might not work as selected class can be not the first in the list.

> Also, what would be the difference between 'begins with' and ' starting with' for the | and ^ filters?

From the documentation:
`|` - The value has to be a whole word, either alone, like class="top", or followed by a hyphen( - ), like class="top-text"! 
`^` - he value does not have to be a whole word!

**Comment by mikezimm** [comment]
@AJIXuMuK , thanks for the explaination :)
